docs/roadmap.md

```diff
@@ -39,11 +43,57 @@ You can help us prioritize by [upvoting existing feature requests](https://githu
 - [ ] Support for injecting request details _(consider not doing globals like Python legacy)_
 - [ ] Support for Router _(multi-file routes)_
 
-#### Improve operational excellence
+#### Service Team co-launches (p1)
+
+This is a catch-all roadmap item that stands in for activities that the team will be doing to support the launch of new features. Powertools for AWS continued partnership with AWS Service Teams involves the team working with product teams from the earliest stages of design to ensure that features are built by working backwards from the customer.
+
+We will announce these features as they are launched and will update this roadmap with more details as they become available.
+
+#### Feature parity (p2)
+
+To close the gap between Powertools for AWS Lambda (Python) and Powertools for AWS Lambda (TypeScript), we will focus our efforts on adding targeted features that are currently missing from the TypeScript version. These include (but are not limited to):
+
+##### Logger
+
+- [ ] [Ability to buffer logs](https://github.com/aws-powertools/powertools-lambda-typescript/discussions/3410){target="_blank"}
+- [ ] [Ability to add a correlation ID to logs via decorator/middleware](https://github.com/aws-powertools/powertools-lambda-typescript/issues/2863){target="_blank"}
+- [ ] [Ability to refresh debug log sampling rate via decorator/middleware](https://github.com/aws-powertools/powertools-lambda-typescript/issues/3278){target="_blank"}
+
+##### Event Handler
+
+In addition to the Event Handler REST feature mentioned above, we will also be working on the following:
+
+- [ ] Implement resolver for Amazon Bedrock Agents
+- [ ] Create RFC for AppSync GraphQL resolver
```

Comment:
Missing Application Load Balancer? In the RFC you mention that ALB will not be supported on day 1.

Reply:
It's not going to be in the developer beta, but we'll add it as part of the initial GA release for Event Handler covered in the first area.